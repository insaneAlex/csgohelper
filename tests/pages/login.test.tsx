import {render} from '@testing-library/react';
import Login from '../../pages/login';

describe('Login', () => {
  it('should render correctly', () => {
    const {container} = render(<Login />);
    expect(container).toMatchSnapshot();
  });
});
