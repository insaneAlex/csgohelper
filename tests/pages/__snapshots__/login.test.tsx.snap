// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render correctly 1`] = `
<div>
  <div
    class="alert"
  >
    Coming soon
  </div>
</div>
`;
